packages/cli/src/commands/store.js

```diff
@@ -1,18 +1,92 @@
 /* global process */
 import fs from 'fs';
 import os from 'os';
+import Buffer from 'buffer';
```

Comment:
Can we use `Uint8Array` instead of `Buffer`?

Reply:
We could, but we have to hand-roll `Buffer.concat` to get there, which does not spark joy.

Reply:
For reference https://evanhahn.com/the-best-way-to-concatenate-uint8arrays/

Reply:
Ultimately sold on hand rolling Buffer.concat by eslint’s insistence that it doesn’t exist.